frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-api.ts

```diff
@@ -7,4 +8,5 @@ export const exposePluginAPI = () => {
     useK8sWatchResources: require('@console/internal/components/utils/k8s-watch-hook')
       .useK8sWatchResources,
   };
+  Object.assign(window.api, exposedAPIs);
```

Comment:
I din't make a new object for it. Should I add it to `window.provisional` instead?